### PR TITLE
handle elibility error

### PR DESCRIPTION
**Pull request by romuloctba**
On Register, there is a feedback when passphrase is created:
image

On Create Pin, if user is not eligible (aka /register errors), error is shown:
image

### CODE REVIEW

File: packages/react-voting-app/src/pages/create-pin/CreatePinPage.tsx [hunk 1]
```diff
@@ -110,11 +116,11 @@ function CreatePinPage(props: CreatePinPageProps) {
     const doRegister = async (hashId: string, publicKey: string) => {
         try {
             setLoading(true)
-            const {vaddrs, vopts} = await Services.Eligibility.register(hashId, publicKey);
-            setVotingOptions(vopts);
-            setVotingAddress(vaddrs);
+            await Services.Eligibility.register(hashId, publicKey);
+            setCanSetPin(true);
```
Comment (ohager) [inline packages/react-voting-app/src/pages/create-pin/CreatePinPage.tsx]: why not `setIsElgible(true)`

**Review by ohager — CHANGES_REQUESTED** [review]

File: packages/react-voting-app/src/pages/create-pin/CreatePinPage.tsx (hunk 1 above)
Comment (romuloctba) [reply packages/react-voting-app/src/pages/create-pin/CreatePinPage.tsx]: because isEligeble is already true when component starts

Comment (romuloctba) [reply packages/react-voting-app/src/pages/create-pin/CreatePinPage.tsx]: and it is set to false when request fails, at the catch(e)

Comment (romuloctba) [reply packages/react-voting-app/src/pages/create-pin/CreatePinPage.tsx]: PR Updated to use `IS_ELIGIBLE_ENUM`

File: packages/react-voting-app/src/pages/create-pin/CreatePinPage.tsx [hunk 2]
```diff
@@ -101,6 +105,7 @@ function CreatePinPage(props: CreatePinPageProps) {
     const passphrase = location.state.passphrase;
     const hashId = location.state.hashId;
     const [loading, setLoading] = useState(false);
+    const [isEligible, setIsEligible] = useState(IS_ELIGIBLE_ENUM.FETCHING);
```
Comment (ohager) [inline packages/react-voting-app/src/pages/create-pin/CreatePinPage.tsx:108]: yep...
being very picky: I prefer CamelCase instead of UPPER_SNAKE

**Review by ohager — APPROVED** [review]

**Pull request merged**